client/gui/CGuiHandler.h

```diff
@@ -115,6 +116,8 @@ class CGuiHandler
 	/// returns current position of mouse cursor, relative to vcmi window
 	const Point & getCursorPosition() const;
 
+	ShortcutHandler & getShortcutsHandler();

```

Comment:
it's non const, then
```suggestion
	ShortcutHandler & shortcutHandler();
```

Reply:
Might turn it into const - right now this class is completely stateless, since it mostly placeholder code for future implementation.

Or rename method to `shortcutHandler` and member with same name - to `shortcutHandlerInstance`?